### ISSUE TITLE
`MSMeasurement.reset_bg` is broken.

### ISSUE THREAD

**Issue by ScottSoren** [ixdat/ixdat#82]
```
>>> msmeas.set_bg(tspan_bg=[0, 10]
>>> msmeas.signal_bgs
{"M2": 2.9e-11, "M4": 2.7e-11, ...}
>>> msmeas.reset_bg()
>>> msmeas.signal_bgs
{"M2": 2.9e-11, "M4": 2.7e-11, ...}  # should be an empty dict!
```
Noticed while looking at #79

**Comment by ScottSoren** [comment]
Fixed in https://github.com/ixdat/ixdat/commit/919ca8eab42c5fbd37eac6b144e18eab8c67d08b

**Comment by KennethNielsen** [comment]
@ScottSoren you wrote that this is fixed, but it is still open. Did you forget to close?

**Comment by ScottSoren** [comment]
Fix is included in ixdat 0.2.4 :)
